main/solution/infrastructure/config/infra/cloudformation.yml

```diff
@@ -202,10 +202,22 @@ Resources:
   # CloudFront
   # =============================================================================================
 
+  ResponseHeadersPolicy:
+    Type: AWS::CloudFront::ResponseHeadersPolicy
+    Properties:
+      ResponseHeadersPolicyConfig:
+        Name: Cache-Control-Response-Headers-Policy
+        Comment: Add Cache-Control header to responses
+        CustomHeadersConfig:
+          Items:
+            - Header: Cache-Control
+              Value: no-store, no-cache
```

Comment:
Add `Cache-Control: no-cache`, it means if we don't cache the content in CDN?

Reply:
It means client side does not cache the content, not CDN.